### ISSUE TITLE
[lcc members portal] Create page to upload private documents on Odoo

### ISSUE THREAD

**Issue by MatSarr** [Lokavaluto/lokavaluto-addons#152]
Some lcc will need official personal documents from the users (IDs, proof of address, KBIS, ...) in order to create digital currency wallet. We need a functionnality that allows the user to upload the requested documents in Odoo.
The documents should be found in the Odoo partner form view linked to user

**Comment by stephansainleger** [comment]
I don't think it is a good idea to add this mandatory field at signup time. Sign up process is only to create a Odoo account, there is no need of CI at that step. 
I suggest to add a page "My ID Documents" in which a user can add its ID card, its proof of address, its KBIS or whatever other documents needed for membership and/or digital currency.
Then if the lcc wants to make these documents mandatory on order to open a digital currency wallet, it can personalize its own Odoo.

**Comment by MatSarr** [comment]
This issue was created before having the all process in head. I agree now with you.
